src/components/ProductTours/ProductTours.jsx

```diff
@@ -42,6 +47,7 @@ const ProductTours = ({ enterpriseSlug }) => {
 
 ProductTours.propTypes = {
   enterpriseSlug: PropTypes.string.isRequired,
+  enableLearnerPortal: PropTypes.bool.isRequired,
```

Comment:
If this component is already connected to the Redux store to get `enterpriseSlug`, should we grab `enableLearnerPortal` from the store versus using prop drilling the value through the parent component?

Reply:
yeah we should do that.